README.md

```diff
@@ -102,6 +102,27 @@ kubectl auth-proxy -n kubetail http://kubetail.svc
 
 Now your computer will automatically open a new browser tab pointing to the kubetail dashboard.
 
+### Option 4: ingress with Helm chart
+
+Set the following in your `values.yaml` replacing where applicable:
+
+```yaml
+ingress:
+  enabled: true
+  annotations:
+    traefik.ingress.kubernetes.io/router.entrypoints: websecure
+    traefik.ingress.kubernetes.io/router.tls: "true"
+    traefik.ingress.kubernetes.io/router.tls.certresolver: le
```

Comment:
I would remove these, not everyone uses traefik, may confuse inexperienced users :)